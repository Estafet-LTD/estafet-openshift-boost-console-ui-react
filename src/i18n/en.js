export default {
  common: {
    name: 'Name',
    displayName: 'Display name',
    environment: 'Environment',
    version: 'Version',
    deployedDate: 'Deployed date',
    view: 'View',
    build: 'Build',
    buildAll: 'Build all',
    testAll: 'Run tests',
    promote: 'Promote',
    promoteAll: 'Promote all',
    goLive: 'Go live!',
    backOut: 'Back out!',
    deployed: 'Deployed {{value}}',
    notDeployed: 'Not deployed',
    lastUpdated: 'Last updated: {{value}}',
    testsFailed: 'Untested or tests failing',
    testsSuccessful: 'Tested successfully',
  },
  features: {
    pageTitle: 'Features',
    dataFallback: 'No features available!',
    searchPlaceholder: 'Search for features',
    status: 'Status: {{value}}',
    waitingSince: 'Waiting since: {{value}}',
  },
  microservices: {
    pageTitle: 'Microservices',
    dataFallback: 'No microservices available!',
    searchPlaceholder: 'Search for microservices',
  },
  microservice: {
    pageTitle: 'Microservice: {{service}}',
    actions: {
      backToList: 'Back to list',
    },
  },
  projects: {
    pageTitle: 'Projects',
    actions: {
      add: 'Add project',
    },
    tableColumns: {
      title: 'Title',
      owner: 'Owner',
      namespace: 'Namespace',
      actions: 'Actions',
    },
  },
};
