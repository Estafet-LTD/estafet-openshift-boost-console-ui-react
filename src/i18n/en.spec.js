import en from './en';

describe('i18n: EN', () => {
  it('should match the snapshot', () => {
    expect(en).toMatchSnapshot();
  });
});
