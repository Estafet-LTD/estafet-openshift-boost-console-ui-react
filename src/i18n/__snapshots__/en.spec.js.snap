// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`i18n: EN should match the snapshot 1`] = `
Object {
  "common": Object {
    "action": Object {
      "pending": "Starting {{action}}...",
      "success": "Successfully started {{action}}!",
    },
    "backOut": "Back out!",
    "build": "Build",
    "buildAll": "Build all",
    "cancel": "Cancel",
    "confirm": "Confirm",
    "delete": "Delete",
    "deployed": "Deployed {{value}}",
    "deployedDate": "Deployed date",
    "displayName": "Display name",
    "edit": "Edit",
    "environment": "Environment",
    "goLive": "Go live!",
    "lastUpdated": "Last updated: {{value}}",
    "name": "Name",
    "notDeployed": "Not deployed",
    "promote": "Promote",
    "promoteAll": "Promote all",
    "submit": "Submit",
    "testAll": "Run tests",
    "testsFailed": "Untested or tests failing",
    "testsSuccessful": "Tested successfully",
    "version": "Version",
    "view": "View",
  },
  "features": Object {
    "dataFallback": "No features available!",
    "pageTitle": "Features",
    "partiallyPromoted": "Feature is partially promoted!",
    "searchPlaceholder": "Search for features",
    "status": "Status: {{value}}",
    "waitingSince": "Waiting since: {{value}}",
  },
  "microservice": Object {
    "actions": Object {
      "backToList": "Back to list",
    },
    "dataFallback": "Microservice data not available!",
    "pageTitle": "Microservice: {{service}}",
  },
  "microservices": Object {
    "dataFallback": "No microservices available!",
    "pageTitle": "Microservices",
    "searchPlaceholder": "Search for microservices",
  },
  "project": Object {
    "actions": Object {
      "backToList": "Back to list",
    },
    "add": Object {
      "pageTitle": "Add project",
      "successMessage": "Successfully added a new project!",
    },
    "edit": Object {
      "pageTitle": "Edit project: {{namespace}}",
      "successMessage": "Successfully edited a project!",
    },
  },
  "projects": Object {
    "actions": Object {
      "add": "Add project",
      "delete": Object {
        "confirm": "Are you sure you want to delete project - {{namespace}}?",
        "pending": "Deleting project {{namespace}}...",
        "success": "Successfully deleted project {{namespace}}!",
      },
    },
    "dataFallback": "No projects available!",
    "pageTitle": "Projects",
    "tableColumns": Object {
      "actions": "Actions",
      "namespace": "Namespace",
      "owner": "Owner",
      "title": "Title",
    },
  },
  "test": Object {
    "simple": "Simple example without params",
    "withParams": "Example with params: {{firstValue}} {{secondValue}}",
  },
}
`;
